src/lib.rs

```diff
@@ -283,6 +285,30 @@ impl<B: Body> Body for http::Response<B> {
     }
 }
 
+impl Body for String {
+    type Data = VecDeque<u8>;
```

Comment:
Noob question from me: what's the benefit of using VecDeque over Bytes or Vec?

Reply:
So we can't use `Vec` since it doesn't impl Buf https://docs.rs/bytes/latest/bytes/trait.Buf.html#impl-Buf-for-VecDeque%3Cu8%3E 

As for Bytes or Vec, I just deferred to Vec since String is a std type and the layout matches more closely to vec. VecDeque has a From<Vec> impl that tries to copy without reallocating. I wasn't sure if this was the case for bytes or not...

cc @seanmonstar since you did some work on bytes last do you know if we should just convert into bytes?

Reply:
Actually I just remembered that axum requires `Data` to be `Bytes` in a bunch of places: 

- [`axum::body::boxed`](https://docs.rs/axum/latest/axum/body/fn.boxed.html)
- [`impl IntoResponse for Response<B>`](https://docs.rs/axum/latest/axum/response/trait.IntoResponse.html#impl-IntoResponse-for-Response%3CB%3E)
- Probably others as well

So I don't think this impl would be very useful for axum users. I'm not sure if thats feasible to change and pretty sure it would be a breaking change 😕

So if going with `Data = Bytes` isn't too bad then I'd much prefer that.